portfolio/src/main/webapp/script.js

```diff
@@ -182,14 +207,14 @@ function createCommentElement(comment) {
 
   const commentHeader = document.createElement('div');
   const nameElement = document.createElement('h3');
-  nameElement.innerText = "Anonymous";
+  nameElement.innerText = String(comment.author);
```

Comment:
Is cast to string necessary here? That's surprising if so

Reply:
I checked an it worked without the cast to string